src/app/screens/LNURLPay.tsx

```diff
@@ -28,6 +28,14 @@ type Props = {
   origin?: Origin;
 };
 
+const Dt = ({ children }: { children: React.ReactNode }) => (
```

Comment:
nit: We do this with `Button` as well, but I feel like naming these as the HTML-tag might be somewhat confusing.  
No need to tackle it here and now. Just a thought.

Reply:
Might just be a react-coding-style-thing I'm not used to though.

Reply:
It's what I'm used to by known UI libs for e.g.: https://mui.com/components/buttons/
But debatable for sure :)

Reply:
No worries, thanks for the link. Might be worth looking at some libs.